lib/lists/lists_page.dart

```diff
@@ -57,11 +57,67 @@ class ListsPage extends ConsumerWidget {
                                   padding: EdgeInsets.all(10),
                                   child: ref.watch(selectedItem) == null
                                       ? Container()
-                                      : JsonViewer(ref.watch(selectedItem)))
+                                      // Calling SwitchJSON widget and passing selectedItem to display
+                                      : SwitchJSON(
+                                          passed_ref: ref.watch(selectedItem)))
                             ],
                           ),
                         ))
                       ])))
                 ])));
   }
 }
+
+// Statefull widget for JSON Viewer, shows data in JSON format of Userfriendly format depending on users choice
+class SwitchJSON extends StatefulWidget {
```

Comment:
Let's have it in a different file and does it have to be StatefulWidget and not ConsumerWidget instead?